<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_up">
    <property name="geometry">
     <rect>
      <x>564</x>
      <y>64</y>
      <width>77</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>上</string>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_left">
    <property name="geometry">
     <rect>
      <x>488</x>
      <y>146</y>
      <width>77</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>左</string>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_down">
    <property name="geometry">
     <rect>
      <x>564</x>
      <y>221</y>
      <width>77</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>下</string>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_right">
    <property name="geometry">
     <rect>
      <x>641</x>
      <y>146</y>
      <width>77</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>右</string>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_quit">
    <property name="geometry">
     <rect>
      <x>679</x>
      <y>340</y>
      <width>91</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>112</y>
      <width>291</width>
      <height>241</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>音频进度</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_music">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>音量</string>
     </property>
    </widget>
    <widget class="QScrollBar" name="horizontalScrollBar_value">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>亮度调节</string>
     </property>
    </widget>
    <widget class="QScrollBar" name="horizontalScrollBar_light">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>33</y>
      <width>255</width>
      <height>51</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="pushButton_paly">
     <property name="text">
      <string>播放</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_suspend">
     <property name="text">
      <string>暂停</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_stop">
     <property name="text">
      <string>停止</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
